README.md

```diff
@@ -23,7 +23,6 @@ Add `plugin:jest-playwright/recommended` to your extends ESLint configuration. F
 ```json
 {
   "extends": ["plugin:jest-playwright/recommended"],
```

Comment:
Let's remove the comma to make this valid JSON.

```suggestion
  "extends": ["plugin:jest-playwright/recommended"]
```

Reply:
My bad, thanks